index.html

```diff
@@ -443,6 +405,8 @@ <h2>Glossary</h2>
 
 <p><dfn class="lint-ignore" data-lt="surrogate">Surrogate code point</dfn>. <a href="https://www.unicode.org/glossary/#surrogate_code_point">Unicode definition</a>: <q>A Unicode code point in the range U+D800..U+DFFF. Reserved for use by UTF-16, where a pair of surrogate code units (a high surrogate followed by a low surrogate) “stand in” for a [=supplementary code point=].</q> This term is also defined by [[INFRA]].</p>
 
+<p><dfn class="lint-ignore export">Surrogate pair</dfn>. In the UTF-16 <a>character encoding</a> of Unicode, a sequence of two <a>surrogate code points</a>, one from the range <code>U+D800...U+DBFF</code> (a <em>high surrogate</em>) followed by one from the range <code>U+DC00...U+DFFF</code> (a <em>low surrogate</em>), encoding a <a>supplementary code point</a>.</p>
```

Comment:
Can we use [=...=] for term refs, rather than `<a>...</a>`?  We use the former elsewhere throughout the glossary.

Reply:
Done.